src/main/java/TaskList.java

```diff
@@ -0,0 +1,35 @@
+//package duke;
+import java.util.*;
+
+public class TaskList {
+
+    ArrayList<Task> tasks;
```

Comment:
I like how you used plural forms for the naming of variables to represent a collection of objects. Good Job!